Cycle/Sources/Cycle/Effect.swift

```diff
@@ -1,26 +1,39 @@
 import Foundation
+import Combine
 
-public struct Effect<A> {
-    public let run: (@escaping (A) -> Void) -> Void
+public struct Effect<Output>: Publisher {
+    public typealias Failure = Never
 
-    public init(run: @escaping (@escaping (A) -> Void) -> Void) {
-        self.run = run
+    private let anyPublisher: AnyPublisher<Output, Never>
+
+    public init(_ publisher: AnyPublisher<Output, Never>) {
+        self.anyPublisher = publisher
+    }
+
+    public func receive<S>(subscriber: S) where S : Subscriber, Failure == S.Failure, Output == S.Input {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/351a3bf30d494055743cf7c8dac1b3f56ce885a2/Cycle/Sources/Cycle/Effect.swift#L13"></span>Colons should be next to the identifier when specifying a type and next to the key in dictionary literals.<br />
<code>colon</code> <code>Effect.swift:13</code></td>
    </tr>
  </tbody>
</table>